angularjs-1.5-basics/gulpfile.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
```

Comment:
__[jshint]__
* Use the function form of "use strict".

<!-- Comment by [SideCI](https://sideci.com) -->